pkg/render/fluentd.go

```diff
@@ -87,15 +87,6 @@ const (
 	LinseedVolumeMountPath = "/var/run/secrets/tigera.io/linseed/"
 	LinseedTokenPath       = "/var/run/secrets/tigera.io/linseed/token"
 
-	probeTimeoutSeconds        int32 = 5
-	probePeriodSeconds         int32 = 5
-	probeWindowsTimeoutSeconds int32 = 10
-	probeWindowsPeriodSeconds  int32 = 10
-
-	// Default failure threshold for probes is 3. For the startupProbe tolerate more failures.
```

Comment:
We should keep this comment and move it to where the startup probe failure threshold is being set so that we know it is purposeful and not an accident or copy/paste setting.

Reply:
This comment is moved to the startup probe.